src/devices/Common/Iot/Device/Multiplexing/samples/OutputSegment-driver.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Same as above. Unless there's a reason, projects should target both netcoreapp2.1 and net5.0

Reply:
This is for the sample app. I'm thinking that samples can all target .NET latest. If you look, you can see that's already the case. For bindings, I totally agree.

Reply:
True, that's fine with me